err_test.go

```diff
@@ -343,6 +343,41 @@ func TestWithClues(t *testing.T) {
 	}
 }
 
+func TestValuePriority(t *testing.T) {
+	table := []struct {
+		name   string
+		err    error
+		expect msa
+	}{
+		{
+			name: "lowest data wins",
```

Comment:
could you please add another test showing that overriding context clues with `With` works as expected? See example below
```go
ctx := clues.Add(context.Background(), "something", 1)
err := clues.NewWC(ctx, "err").With("in-err", 2, "something", 2)

// expect 2, 2
```